manifests/concourse-manifest/github_auth/config.yml

```diff
@@ -14,10 +14,9 @@ instance_groups:
                 users:
                 - 46bit
                 - AP-Hunt
-                - chrisfarms
                 - LeePorte
-                - paroxp
                 - richardTowers
                 - rhowe-gds
                 - tlwr
                 - tnwhitwell
+                - mogds
```

Comment:
Any chance we can keep this alphabetically sorted?

Reply:
Oops, yep! I didn't even spot that it was